script/render-schemas.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Would it be possible to move this to go code? One option is to use go code generation. Another is to use `embed` in `otelx` and expose the schema as a global variable, that you then import here (see @zepatrik suggestion)

Reply:
Bear with me because I'm new to this, but aren't we doing so already? See:
- https://github.com/ory/x/blob/999103ce50c0c88f193202c0371437eaeeb43fec/otelx/config.go#L28
- https://github.com/ory/x/blob/999103ce50c0c88f193202c0371437eaeeb43fec/configx/schema.go#L31

Hence, the otelx schema is already being loaded.
As I understand it, Patrik's suggested script is to simply replace the "internal" schema ref (`ory://tracing-config`) to one that is accessible "globally" (i.e. using the absolute URL).

Reply:
Ah yes, perfect :) So we can remove this file then?

Reply:
We need this for schemas to be consumed externally, like through schemastore.

Reply:
Ah, I see. In that case a bit of documentation what this script's purpose is and how to use it would be appreciated. It's also a bit confusing now, because we have two identical config schemas and it's not clear what's what, especially since the newly generated schema lives now under the location where the old schema was stored previously. Is there a better way to publish this for schemastore?

Reply:
We can put it anywhere we want to. The new schema is now at `.schema/config.schema.json`, while the internal one is still under `embedx`. Any consuming go service would still use the one from `embedx`, but it also has to register the dependency schemas. That's why there should be a helper function that does exactly that, basically the same as https://github.com/ory/keto/blob/273b3eada1b376d4b09a0f286d87a01e2d619ef8/embedx/schemas.go#L28-L40.

I'll write something better in go later, but did not have time yet. This was only a hack to get it working.

Reply:
We could then also organize the various openapi specs properly. Not sure what the best naming would be for both of those assets.

Reply:
Folks, so what's the resolution on this? Shall I just plop it into `.schemastore/`?

Reply:
Yes? :)